accounting/transferwise/tasks.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+#
+# OpenCraft -- tools to aid developing and hosting free software projects
+# Copyright (C) 2017-2018 OpenCraft <contact@opencraft.com>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+"""
+TransferWise tasks.
+"""
+
+from django.conf import settings
+from django.core.mail.message import EmailMessage
+from huey import crontab
+from huey.contrib.djhuey import db_periodic_task
+
+from accounting.transferwise import models
+
+
+@db_periodic_task(crontab(day=settings.TRANSFERWISE_BULK_PAYMENT_DAY, hour="0", minute="0"))
+def generate_bulk_payment_csv():
```

Comment:
@UmanShahzad It would be nice to continue the naming scheme established in `accounting/invoice/tasks.py`, and call this `send_bulk_payment_csv`.

Reply:
That's probably a good idea, yeah.